code/game/machinery/transformer.dm

```diff
@@ -62,7 +62,6 @@
 	playsound(src.loc, 'sound/items/Welder.ogg', 50, 1)
 	H.emote("scream") // It is painful
 	H.adjustBruteLoss(max(0, 80 - H.getBruteLoss())) // Hurt the human, don't try to kill them though.
-	H.handle_regular_hud_updates() // Make sure they see the pain.

```

Comment:
wait, really? REALLY?!

Old code makes me gag sometimes.

Reply:
You can be sure we've all seen the pain mystery oldcoder